website/docs/how-to/how-to-troubleshoot-flag-not-returned.md

```diff
@@ -0,0 +1,14 @@
+---
+title: My feature flag is not returned in the Frontend API/Edge/Proxy
```

Comment:
This is only when using flags in the client SDK / front-end SDKs, right? If using Edge with a back-end SDK, you'll still get disabled flags

Reply:
Yes, but then you will not have this problem :)

The reasoning is that if you are here, it is because the feature is not being returned, not because your feature is being returned as disabled. So, I'm not sure it is worth clarifying that this is only in client/frontend SDKs...

Reply:
Ah, I see your reasoning. Maybe add a note in the text that this only applies to front-end/client-side SDKs/endpoints?

Reply:
I think a note might be confusing, if you have this problem using a server-side endpoint (and I believe you shouldn't have it), we should update this guide. It would be an unknown unknown.

Reply:
Got it. I'll leave it up to you ☺️

Reply:
I believe this troubleshooting guide will be a bit different than regular documentation:
1. More brief and linking to the source of truth in our docs
2. Generic, on the assumption that people landing on this page have the problem, and if we don't have the solution it's probably something we didn't know.

I'm no expert, so I'll keep relying on you :hugs:

Reply:
Agreed! Thanks again for the contribution 🙏🏼